Sources/NIO/Docs.docc/index.md

```diff
@@ -0,0 +1,183 @@
+# ``NIO`` <!-- SwiftNIO -->
```

Comment:
Unrelated: should be doc be updated to cover async/await support?

Reply:
good idea,probably follow on PR or commits? this is all taken from the current readme

Reply:
> probably follow on PR or commits?

yeah